doc/user/content/_index.md

```diff
@@ -9,14 +9,9 @@ weight: 1
 
 # Materialize documentation
 
-Materialize is the Operational Data Warehouse that delivers the speed of
-streaming with the ease of a data warehouse.
-
-Using SQL and common tools in the wider data ecosystem, Materialize allows you
-to build real-time automation, engaging customer experiences, and interactive
-data products that drive value for your business while **reducing the cost of
-data freshness**.
-
+Materialize is the Cloud Operational Data Store that delivers the speed of
+streaming with the ease of a data warehouse. With Materialize, organizations can
+use SQL to transform, deliver, and act on fast-changing data.
```

Comment:
```suggestion
Materialize is the Cloud Operational Data Store that lets you use SQL to
transform, deliver, and act on fast-changing data.
```

Reply:
Checked with Nate and he suggested we drop "the speed of streaming with the ease of a data warehouse", in line with the [new messaging](https://docs.google.com/document/d/1K4hGwuTtIYutLJMIlVXjZgSi-Uscuegvqnh39WFPLN4/edit).